gateway-provider-security-webappsec/src/main/java/org/apache/knox/gateway/webappsec/deploy/WebAppSecContributor.java

```diff
@@ -55,6 +55,10 @@ public class WebAppSecContributor extends ProviderDeploymentContributorBase {
   private static final String RATE_LIMITING_PREFIX = "rate.limiting";
   private static final String RATE_LIMITING_SUFFIX = "_RATE.LIMITING";
   private static final String RATE_LIMITING_ENABLED = RATE_LIMITING_PREFIX + ".enabled";
+  private static final String SECURITY_HEADER_PREFIX = "security.header";
```

Comment:
I think the prefix should be `"security.header."` (note the `.` at the end).

Reply:
I agree but others are this way too just keeping it consistent.

Reply:
It is already inconsistent. I am going to follow up with another change make them consistent.